patches/0005-Add-OpenSSL-crypto-module.patch

```diff
@@ -1,20 +1,183 @@
 From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
-From: Quim Muntal <qmuntaldiaz@microsoft.com>
-Date: Wed, 22 Jun 2022 12:31:24 +0000
+From: qmuntal <qmuntaldiaz@microsoft.com>
+Date: Thu, 30 Jun 2022 10:06:19 +0200
 Subject: [PATCH] Add OpenSSL crypto module
```

Comment:
I think you mentioned in Go sync changing the patch names to call these "crypto backends" rather than modules?